### PR TITLE
blog: add cli v0.17.1.6

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: _posts/2020-12-07-monero-0.17.1.6-released.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+monero-linux-x86-v0.17.1.6.tar.bz2, 300e7608927867d63765704a19baa90366b5897e3cef8a56da29ae3a6a5b97a3
+monero-linux-armv8-v0.17.1.6.tar.bz2, 874d3de908fb4301de19301b928a1c477a883c40b2491b3b3193df99561a8904
+monero-linux-armv7-v0.17.1.6.tar.bz2, 018270d8dde8e895fdc7b5b6de95c36b7e7a63d46406339f810b6fdf91e0b8bc
+monero-android-armv8-v0.17.1.6.tar.bz2, 0dc3cc265ae0365cb927c235f9ba4391f3c2be7043d183769de5b9b97736b359
+monero-android-armv7-v0.17.1.6.tar.bz2, 2ab2e4c715a3978ed36c55848313f8a252b1f7141910c0bcbd52070fcaefcff1
+monero-freebsd-x64-v0.17.1.6.tar.bz2, 1ad59103c9ea3a2256c8f4b7066cefd7ce2ebd2b52360edb539115c80a7e6ee1
+```
+
+A GPG-signed list of the hashes is at [https://getmonero.org/downloads/hashes.txt](https://getmonero.org/downloads/hashes.txt) and should be treated as canonical, with the signature checked against the appropriate GPG key in the source code (in /utils/gpg_keys). To ensure that the files you download are those originally posted by the maintainers, you should both check that the hashes of your files match those on the signed list, and that the signature on the list is valid.
```
Comment (erciccione) [inline _posts/2020-12-07-monero-0.17.1.6-released.md:73]: Better use this structure from now on:

```suggestion
A GPG-signed list of the hashes is at [{{ site.baseurl_root }}/downloads/hashes.txt](https://getmonero.org/downloads/hashes.txt) and should be treated as canonical, with the signature checked against the appropriate GPG key in the source code (in /utils/gpg_keys). To ensure that the files you download are those originally posted by the maintainers, you should both check that the hashes of your files match those on the signed list, and that the signature on the list is valid.
```

Comment (erciccione) [reply _posts/2020-12-07-monero-0.17.1.6-released.md:73]: Fixed the other occurrences with #1361

Comment (selsta) [reply _posts/2020-12-07-monero-0.17.1.6-released.md]: updated

**Review by erciccione — APPROVED** [review]

**Review by erciccione — APPROVED** [review]
Hashes to be verified

**Comment by binaryFate** [comment]
Hashes ok

**Pull request merged**